tests/test_connector.py

```diff
@@ -343,6 +343,28 @@ def test_release_close(loop):
     assert proto.close.called
 
 
+def test__drop_acquire_per_host1(loop):
```

Comment:
@asvetlov is this `double__underscore` notation intentional?

Reply:
Yes. It is a test for `_drop_acquire_per_host` method.